### DISCUSSION TITLE
v3.18.1

### DISCUSSION THREAD

**Discussion by mehalter** [AstroNvim/AstroNvim#1964] [Announcements]
### Bug Fixes

* **resession:** improve resession buffer loading ([0d880a0](https://github.com/AstroNvim/AstroNvim/commit/0d880a07029804778b7415919ca84b340e445ca8))

**Full Changelog**: https://github.com/AstroNvim/AstroNvim/compare/v3.18.0...v3.18.1

<hr /><em>This discussion was created from the release <a href='https://github.com/AstroNvim/AstroNvim/releases/tag/v3.18.1'>v3.18.1</a>.</em>

**Comment by hniia** [comment]
![image](https://github.com/AstroNvim/AstroNvim/assets/108707880/8b05489b-c276-40a3-9e2f-bdee19cce99b)
 What fonts I need to install to fix this ?
 I'm using nvim over ssh

**Reply by mehalter** [reply]
A font patched with nerd fonts v13+
